sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Allow `RetryStrategy`s to specify a delay and redirect URI at the same time. [#23269](https://github.com/Azure/azure-sdk-for-js/pull/23269)
```

Comment:
could you please bump the minor version, so in this case the version should be 1.10.0.

Reply:
Done! I was on the fence about bumping core-rest-pipeline considering the API doesn't change, so thanks for the clarification!